requirements.txt

```diff
@@ -12,7 +12,7 @@ pandas>=1.1.3,<2.2; python_version == "3.9"
 pandas>=1.3.0,<2.2; python_version >= "3.10"
 pandas<2.2; python_version >= "3.12"
 # analytics
-posthog>=2.1.0,<3
+posthog==3.7.4
```

Comment:
Are we OK pinning to a specific version here or do we want to keep this more relaxed like other GX Core deps? Maybe just change the `<3` to `<4`?

Reply:
Yeah, agreed it's best to keep it loose to accommodate more GX users. I'd only go with specific versions when we can control the environment in which the deps are installed.